nextcord/state.py

```diff
@@ -1233,8 +1234,8 @@ async def _delay_ready(self) -> None:
             pass
         else:
             # dispatch the event
-            self.call_handlers("ready")
-            self.dispatch("ready")
+            self.call_handlers("ready")  # TODO: Figure out wtf this is? Is this the parsers?
```

Comment:
https://github.com/nextcord/nextcord/blob/22eb5d169f6a05f2c813d72a158972a0ba3d7dc1/nextcord/client.py#L375
I believe this is used so overriding `on_ready` does not clash with the `_ready` `asyncio.Event`

Reply:
I understand now, thank you!